crates/artifacts/zksolc/src/lib.rs

```diff
@@ -97,7 +97,25 @@ pub struct EraVM {
     /// The contract bytecode.
     /// Is reset by that of EraVM before yielding the compiled project artifacts.
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub bytecode: Option<BytecodeObject>,
+    bytecode: Option<BytecodeObject>,
```

Comment:
iirc if you remove `pub` serde will not encode/decode it.

Reply:
This is the output of the compiler, tests still work with it (and also an example repo I have open does)